src/Livewire/SlowQueries.php

```diff
@@ -56,7 +56,10 @@ public function render(): Renderable
         return View::make('pulse::livewire.slow-queries', [
             'time' => $time,
             'runAt' => $runAt,
-            'config' => Config::get('pulse.recorders.'.SlowQueriesRecorder::class),
+            'config' => [
+                'highlighting' => true,
+                ...Config::get('pulse.recorders.'.SlowQueriesRecorder::class),
+            ],
```

Comment:
I've done this so that we don't break existing installs. We could remove it once we tag a 1.0 if we tell everyone to re-publish the config.